Explaining how to opt in modules (#10991)
ARMmbed/mbed-os: features/cryptocell/FEATURE_CRYPTOCELL310/Readme.md

1. Add your CC setup code:
	* Implement `crypto_platform_setup()` and `crypto_platform_terminate()` to enable CC on your platform, in case you have board-specific setup functionality, required for CC setup. You MUST call 'SaSi_LibInit()` and 'SaSi_LibFini()' respectively in these functions.
	* Define `crypto_platform_ctx` in `crypto_device_platform.h` in a way that suits your implementation.

## Enabling optional alternative drivers

> There are three additional modules that have alternative implementation support, which are not enabled by default.

Reviewer: ```suggestion
There are three additional modules with alternative implementation support, disabled by default.
```

The reason is to allow backwards compatability, as these modules don't have full functionality, and return `MBEDTLS_ERR_PLATFORM_FEATURE_UNSUPPORTED` for some features.
The modules are:

> * `AES` which only supports 128 bit key size, in opposed to previous suipport for all key sizes.

Reviewer: `support` typo

Author: done
